azavea/raster-foundry
# Add support for database migrations

This also creates models for users and organizations (generated automatically from the db schema).

## Testing
 - Open up an `sbt` console: `./scripts/console app-server ./sbt`
 - Initialize your database for migrations: `mg init`
 - Run migrations: `~mg migrate` (hit `ENTER` when complete)
 - Start the server: `app/run`
 - Hit the healthcheck endpoint to verify everything worked: `http://localhost:9000/healthcheck`
   - The healthcheck now checks for the existence of at least one user in the database, so if a status of `OK` is received, it means the migrations were successful.
